ale_linters/scala/scalastyle.vim

```diff
@@ -0,0 +1,46 @@
+" Author: Kevin Kays - https://github.com/okkays
+" Description: Support for the scalastyle checker.
+
+let g:ale_scala_scalastyle_options =
+\   get(g:, 'ale_scala_scalastyle_options', '-c scalastyle-config.xml')
```

Comment:
Yup, although looking into it, it seems scalastyle.org defaults to `scalastyle_config.xml` while something (possibly sbt) has caused other projects to use `scalastyle-config.xml`.  I've tried to account for both.